neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
A navíc koukám, že v `build_model` hláška je, zrovna mi to na ní záhadně padá a jsem rád, že mi to přehledně říká kde a nemusím to luštit z tracebacku.